packages/@uppy/companion/src/server/provider/zoom/index.js

```diff
@@ -157,6 +152,7 @@ class Zoom extends Provider {
     })
   }
 
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
This indicates that we could move it out of the class (i.e. as a standalone function). Can we do that in this PR?

Reply:
Right, but this has to be done in all providers so I didn't feel it was worth the effort (and breakage of git blame/history). maybe a todo will do?

Reply:
I just tried to refactor one of the provider classes by moving it out of the class, but then we have to add 6 new `// eslint-disable-next-line class-methods-use-this` because each instance method gets the same error (due to no longer using `this.#withErrorHandling`)